patches/minecraft/net/minecraft/world/level/storage/loot/LootTable.java.patch

```diff
@@ -101,3 +124,11 @@
              jsonobject.add("pools", p_79179_.serialize(p_79177_.f_79109_));
           }
  
+          if (!ArrayUtils.isEmpty((Object[])p_79177_.f_79110_)) {
+             jsonobject.add("functions", p_79179_.serialize(p_79177_.f_79110_));
++         }
++         if (!ArrayUtils.isEmpty(p_79177_.conditions)) {
++            jsonobject.add("forge:conditions", net.minecraftforge.common.crafting.CraftingHelper.serialize(p_79177_.conditions));
```

Comment:
While this works fine since the member name for conditions in the loot table manager cannot be modified otherwise, I dislike it being hardcoded here. Could we perhaps have a constant somewhere and reference that instead?